cli/src/main.rs

```diff
@@ -37,6 +44,13 @@ struct Args {
     /// reduced if bigger than the biggest block in the database.
     #[arg(long)]
     end_block: u64,
+
+    /// The filename of the histogram SVG image.
+    ///
+    /// If `None`, histogram generation is skipped.
+    /// If the file exists already, it tries overwriting.
```

Comment:
I think it should report an error by default if the file exists. If users want it to overwrite, there should be an additional `--overwrite` flag.